src/app/helpers/storybook/ampDecorator/index.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { ampScript, ampNoscript } from '../../../lib/constants/styles';
+
+/*
+  Contains AMP required markup https://www.ampproject.org/docs/fundamentals/spec#required-markup
+*/
+
+const AmpDecorator = storyFn => (
```

Comment:
We should pass the components through as `children`

```suggestion
const AmpDecorator = children => (
```

Reply:
As per the other comment I think its misleading for it to be children as thats a react-y convention